packages/code-du-travail-frontend/src/search/SearchResults.js

```diff
@@ -40,14 +40,18 @@ const ResultItem = withRouter(({ _id, _source, highlight, router }) => {
   const excerpt = makeExcerpt(highlight);
 
   const route = getRouteBySource(_source.source);
+  const anchor = _source.anchor ? _source.anchor.slice(1) : "";
 
   // internal links
   if (route) {
     return (
       <li className="search-results__item">
         <Link
-          route={route}
-          params={{ slug: _source.slug, q: router.query.q, search: 0 }}
+          href={{
+            pathname: `${route}/${_source.slug}`,
+            hash: anchor,
+            query: { q: router.query.q, search: 0 }
+          }}
```

Comment:
si je ne me trompe pas tu peux utiliser les "named routes" et le `Link` de `next-routes` :

```js
<Link route={route} params={{
    slug: _source.slug,
    q: router.query.q,
    search: 0
}}>
```

Reply:
pros : you dont need to build the url, you only pass named routes + params.

wdyt ?

Reply:
> coté front-end, j'utilise les Link et Router.push de next et non ceux next-routes car il ne supporte pas les hashdans les url

Reply:
en gros le wrapper ne permet pas de passer un objet a `Link` / `Router.push`mais seulement une `String` or le hash est supporté dans le cas où tu leur passes un objet

Reply:
mmm en effet ! https://github.com/fridays/next-routes/pull/91

on va essayer de reveiller le maintainer de next-routes :)